manifesto/personas.md

```diff
@@ -0,0 +1,63 @@
+# Kyma Personas
+
+The following personas are fictional characters, which we defined in order to represent the different user types that might use Kyma. Keep in mind those personas when contributing to Kyma as they will help you to understand Kyma users' needs and goals (GAINS :green_heart:) and major pain points (PAINS :broken_heart:).
+
+1. [Matthew](#matthew---cloud-native-software-engineer)
+2. [Nicolas](#nicolas---tech-lead)
+
+## Matthew - Cloud Native Software Engineer
+
+![Mathew](assets/matthew.png)
+
+|                             |                                                                                                 |
+| --------------------------- | ----------------------------------------------------------------------------------------------- |
+| Name                        | **Matthew**                                                                                     |
+| Age                         | 26                                                                                              |
+| Ocupation                   | Cloud Native Engineer at **PureJoy**                                                            |
+| Characteristic              | Open source ethusiast and contributor. You will meet him at the webinars and community meetups. |
+| K8S experience              | :book: :book: :book: :book: :book: :book:                                                       |
+| Accountability/Decisiveness | :sweat: :sweat:                                                                                 |
```

Comment:
What is the rage for `K8S experience` and `Accountability/Decisiveness`? I was wondering whether it's (1-6) or even (1-10)
Could this range be added to the first column?

Could also 💪  be used instead of 😓  for grading `Accountability/Decisiveness` ?
```suggestion
| K8S experience (1-6)  | :book: :book: :book: :book: :book: :book:                                                       |
| Accountability/Decisiveness (1-6) | 💪  💪                                                                                  |
```